### ISSUE TITLE
[Accessibility] The "Location" Filter is inaccessible using the keyboard only

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4963]
## Describe the bug

* The Location filter in the "Map View" is not keyboard-focusable and operable.Users are unable to focus and write on the input the location using the keyboard only.

## To Reproduce

1. Launch the local host
2. Navigate to the "Map View"
3. Find the location filter under the Map View and try to interact with it without using the mouse.
4. Observe if you are able to focus on that drop down as well as inputing the location without using the mouse

## Expected behavior

Location filter should be keyboard-focusable and operable, allowing users to interact with it

## Screenshots

![Screenshot from 2024-06-18 18-16-59](https://uploads.linear.app/f1df6e34-3bca-4ade-bb48-e6a457e64091/6aabc719-1996-4094-a268-9da429138e0c/72dce2c5-6730-4f64-8cb8-7446fbf0051c?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2YxZGY2ZTM0LTNiY2EtNGFkZS1iYjQ4LWU2YTQ1N2U2NDA5MS82YWFiYzcxOS0xOTk2LTQwOTQtYTI2OC05ZGE0MjkxMzhlMGMvNzJkY2UyYzUtNjczMC00ZjY0LThjYjgtNzQ0NmZiZjAwNTFjIiwiaWF0IjoxNzE5MzEzMDg1LCJleHAiOjMzMjg5ODczMDg1fQ.Ss4j7A2cqFAkX78oq-uGyKqF5qGlTaf3MHwiBcI8ax8)

**Comment by Angamanga** [comment]
Test linear-integration

**Comment by Angamanga** [comment]
Responding

**Comment by Angamanga** [comment]
@rosekivuvan Sorry, I was just testing for myself 😀 You don't need to do anything!

**Comment by Angamanga** [comment]
Closing but keeping in Linear: https://linear.app/ushahidi/issue/USH-1439/[accessibility]-the-location-filter-is-inaccessible-using-the-keyboard
